assets/src/edit-story/components/form/advancedDropDown/dropDown.js

```diff
@@ -58,24 +58,27 @@ const Container = styled.div`
  * @param {string} props.dropDownLabel The visible label of the dropdown select.
  * @return {*} Render.
  */
-function DropDown({
-  onChange,
-  disabled = false,
-  selectedId,
-  options,
-  hasSearch = false,
-  getOptionsByQuery,
-  onObserve,
-  primaryOptions,
-  primaryLabel,
-  priorityOptions,
-  priorityLabel,
-  searchResultsLabel,
-  renderer,
-  isInline = false,
-  dropDownLabel = '',
-  ...rest
-}) {
+const DropDown = forwardRef(function DropDown(
+  {
+    onChange,
+    disabled = false,
+    selectedId,
+    options,
+    hasSearch = false,
+    getOptionsByQuery,
+    onObserve,
+    primaryOptions,
+    primaryLabel,
+    priorityOptions,
+    priorityLabel,
+    searchResultsLabel,
+    renderer,
+    isInline = false,
+    dropDownLabel = '',
+    ...rest
+  },
+  ref
```

Comment:
needed to pass the ref in from the textStyle panel so that we can focus the font dropdown when the quick action menu for editing text is selected.